tools/docker-compose-base.yml

```diff
@@ -42,6 +42,12 @@ services:
       API_HF_TIMEOUT_SECONDS: ${API_HF_TIMEOUT_SECONDS-0.2}
       API_MAX_AGE_LONG: ${API_MAX_AGE_LONG-120}
       API_MAX_AGE_SHORT: ${API_MAX_AGE_SHORT-10}
+      CACHED_ASSETS_S3_BUCKET: ${CACHED_ASSETS_S3_BUCKET-}
```

Comment:
We generally repeat the default value here. I'm not sure if it's the best way to do it, but we should try to stay consistent, so: add them here, or remove them from elsewhere, depending on what seems more logical